src/main/java/com/celements/rights/access/RightsAccessScriptService.java

```diff
@@ -27,7 +27,7 @@ public EPubUnpub getEPubUnpub(String name) {
   }
 
   public EAccessLevel getEAccessLevel(String xwikiRight) {
-    return EAccessLevel.getAccessLevel(xwikiRight);
+    return EAccessLevel.getAccessLevel(xwikiRight).get();
```

Comment:
@msladek I used here: return EAccessLevel.getAccessLevel(xwikiRight).orNull(); which checks if it is present and gets it, if not present it returns null.